easybuild/easyconfigs/d/DeepSurv/DeepSurv-20180922-fosscuda-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,66 @@
+easyblock = 'PythonBundle'
+
+name = 'DeepSurv'
+# the latest stable release, DeepSurv-2.0.0 is not fully compatible with Python 3
+version = '20180922'
```

Comment:
@smoors Is it worth the effort to try and get a new version tagged for this latest commit, by asking the developers?

Otherwise we'll adopt an alternate versioning scheme, which may be annoying later...

Alternatively, we can consider using `-20180922` in the versionsuffix and sticking to the `2.0.0` version (depending on how much other changes were made, I guess)

Reply:
@boegel the developers did not respond to my request:: https://github.com/jaredleekatzman/DeepSurv/issues/50
do you want to use the alternative versioning that you suggested?. the last release (2.0.0) is from Dec 30, 2017...

Reply:
I ended up adding the date to the 2.0.0 version